test/apns.provider.test.js

```diff
@@ -24,7 +34,7 @@ describe('APNS provider', function() {
     afterEach(mockery.tearDown);
 
     it('sends Notification as an APN message', function(done) {
-      givenProviderWithConfig();
+      givenProviderWithConfig(defaultConfiguration);
```

Comment:
The idea of `givenProviderWithConfig` is to create a provider with sensible default configuration, so that we don't have to repeat ourselves in the tests and also don't have to change all tests when the default configuration changes like here.

Please change `givenProviderWithConfig` to set sensible `apns` defaults and revert changes in this file.